### PR TITLE
packet 0.2.2

### PR DESCRIPTION
**Pull request by luthermonson**
https://github.com/rancher/rancher/issues/23476

### CODE REVIEW

**Comment by deniseschannon** [comment]
@luthermonson did you try testing out if packet still works after this change?

**Comment by luthermonson** [comment]
do not merge this until UI and testing is done, this is just a PR to get us the new binary/shasum

File: app/machinedriver_data.go [hunk 1]
```diff
@@ -99,8 +99,8 @@ func addMachineDrivers(management *config.ManagementContext) error {
 		"", "3f793ebb0ebd9477b9166ec542f77e25", nil, false, false, management); err != nil {
 		return err
 	}
-	if err := addMachineDriver(PacketDriver, "https://github.com/packethost/docker-machine-driver-packet/releases/download/v0.1.4/docker-machine-driver-packet_linux-amd64.zip",
-		"", "2cd0b9614ab448b61b1bf73ef4738ab5", []string{"api.packet.net"}, false, false, management); err != nil {
+	if err := addMachineDriver(PacketDriver, "https://github.com/packethost/docker-machine-driver-packet/releases/download/v0.2.2/docker-machine-driver-packet_linux-amd64.zip",
+		"", "e03c6bc9406c811e03e9bc2c39f43e6cc8c02d1615bd0e0b8ee1b38be6fe201c", []string{"api.packet.net"}, false, false, management); err != nil {
```
Comment (deniseschannon) [inline app/machinedriver_data.go]: Can we add this in as their UI component? 

https://tinkerbell.org/ui-driver-packet/1.0.1/component.js

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/23279

**Pull request closed without merging**